node/mock-test.ts

```diff
@@ -53,7 +57,7 @@ export class MockTestRunner {
         return this.stderr.indexOf(message) > 0;
     }
 
-    public run(nodeVersion?: number): void {
+    public async run(nodeVersion?: number): Promise<void> {
```

Comment:
This looks breaking. We should keep backward compatibility with previous api

Reply:
done